lib/src/core/context.dart

```diff
@@ -231,27 +237,34 @@ class ReactiveContext {
         return true;
 
       case DerivationState.possiblyStale:
-        return untracked(() {
-          for (final obs in derivation._observables) {
-            if (obs is ComputedValue) {
-              // Force a computation
+        final prevDerivation = untrackedStart();
```

Comment:
Didn't changing this from a callback just make the code more error prone, as you have to remember to call `untrackedEnd` before every return? Use try-finally at least

Reply:
Agree @katis. I brought it back to the callback version of `untracked`